index.html

```diff
@@ -11,34 +11,217 @@
     <main>
         <header class="header-container"> 
                 <nav class="navbar">
-                    <a class="logo" href="./index.html">John Doe</a>       
-                    <img src="./img/hearder img/Union.png" alt="toggle-icon">
+                    <a class="name" href="./index.html">John Doe</a>       
+                    <img class="logo" src="./img/hearder img/Union.png" alt="toggle-icon">
                 </nav>
         </header>
         <section class="sec-container">
             <div class="hero">
-                <h1>Hey There. I’m Alex</h1>
-                <h4>I’am a Software Developer</h4>
+                <h1 class="header">Hey There. I’m Bawa</h1>
+                <h4 class="sub-header">I’am a Software Developer</h4>
                 <p class="para">
                    I can help you build a product , feature or website Look through some of my work 
                    and experience! If you like what you see and have a project you need coded, 
                    don’t hestiate to contact me.
                </p>
-               <div>
                 <ul class="anchor">
                    <li><a href="#"><img class="social" src="./img/social icons/Icon -GitHub.png" alt="-GitHub-icon"></a></li>
                    <li><a href="#"><img class="social" src="./img/social icons/Icon - Linkedin.png" alt="Linkedin-icon"></a></li>
                    <li><a href="#"><img class="social" src="./img/social icons/Icon - Angelist.png" alt="Angelist-icon"></a></li>
                    <li><a href="#"><img class="social" src="./img/social icons/Icon - Twitter.png" alt="Twitter-icon"></a></li>
                    <li><a href="#"><img class="social" src="./img/social icons/Icon - Medium.png" alt="Medium-icon"></a></li>
                 </ul>   
-               </div> 
             </div>
         </section>
-
         
-      
 
+        <div class="main-container">
```

Comment:
Please consider replacing the `div tag` with the `section tag` to make it semantically correct 👍